src/helper/batchTransfer.ts

```diff
@@ -19,7 +19,13 @@ const makeVestedConfig = (chain: ChainType, reward: BigNumber): VestingConfig =>
       // 1 month = 403200 blocks
       return {
         srcAddress: 'aXNWfAMUV3YjRoGgceJJpieqzteL4jUWR7LM4xZfHfCGDfQ',
-        perBlock: reward.div(3628800).toFixed(18).toString(),
+        perBlock: reward.div(3628800).toFixed(15).toString(),
+        startingBlock: 3628800,
+      };
+    case 'shibuya':
+      return {
+        srcAddress: 'aXNWfAMUV3YjRoGgceJJpieqzteL4jUWR7LM4xZfHfCGDfQ',
+        perBlock: reward.div(3628800).toFixed(15).toString(),
```

Comment:
if we are going to use this script for the Polkadot auction as well, I suggest storing `3628800` (`startingBlock`) as a configurable constant.
The token decimal point (`.toFixed(15)`) should be configurable as well.

Reply:
Thanks. It uses constantly.